docs/faq.rst

```diff
@@ -27,7 +27,13 @@ Can I copy or synchronize my repo to another location?
 ------------------------------------------------------
 
 Yes, you could just copy all the files. Make sure you do that while no
-backup is running (use `borg with-lock ...`). So what you get here is this:
+backup is running. If you copy a repository while backup is running 
+lock held will be present in the copy. Thus, before using borg on the copy 
```

Comment:
If you copy a repository while a backup is running, (the lock held | a lock) will be present in the copy.